docs/framework/whats-new/index.md

```diff
@@ -30,23 +31,107 @@ This article does not provide comprehensive information about each new feature a
 
-<a name="v47"></a> 
-## Introducing the .NET Framework 4.7
+<a name="v471"></a> 
+## Introducing the .NET Framework 4.7.1
+
+The .NET Framework 4.7.1 builds on the .NET Framework 4.6, 4.6.1, 4.6.2, and 4.7 by adding many new fixes and several new features while remaining a very stable product.
+
+### Downloading and installing the .NET Framework 4.7.1
+ 
+You can download the .NET Framework 4.7.1  from the following locations:
+
+- [.NET Framework 4.7.1 Web Installer](http://go.microsoft.com/fwlink/?LinkId=852095)
+
+- [NET Framework 4.7.1 Offline Installer](http://go.microsoft.com/fwlink/?LinkId=852107)
+
+The .NET Framework 4.7.1 can be installed on Windows 10, Windows 8.1, Windows 7 SP1, and the corresponding server platforms starting with Windows Server 2008 R2 SP1. You can install the .NET Framework 4.7.1 by using either the web installer or the offline installer. The recommended way for most users is to use the web installer.
+
+You can target the .NET Framework 4.7.1 in Visual Studio 2012 or later by installing the [.NET Framework 4.7.1 Developer Pack](http://go.microsoft.com/fwlink/?LinkId=852105). 
+
+### What's new in the .NET Framework 4.7.1
+
+The .NET Framework 4.7.1 includes new features in the following areas:
+ 
+- [Core](#core471)
+- [Common language runtime (CLR)](#clr)
+- [Networking](#net471)
+- [ASP.NET](#asp-net471) 
+
+In addition, a major focus in the .NET Framework 4.7.1 is improved accessiblity, so that apps are able to work well with a variety of devices and form factors. For information on accessibility improvements in the .NET Framework 4.7.1, see [What's new in accessibility in the .NET Framework](whats-new-in-accessibility.md). 
+
+<a name="core471" />
+#### Core
+
+**Support for .NET Standard 2.0**
+
+[.NET Standard](~/docs/standard/net-standard.md) defines a set of APIs that must be available on each .NET implementation that supports that version of the standard. The .NET Framework 4.7.1 fully supports .NET Standard 2.0 and adds [about 200 APIs](https://github.com/dotnet/standard/blob/master/netstandard/src/ApiCompatBaseline.net461.txt) that are defined in .NET Standard 2.0 and are missing from the .NET Framework 4.6.1, 4.6.2, and 4.7. (Note that these versions of the .NET Framework support .NET Standard 2.0 only if additional .NET Standard support files are also deployed on the target system.) For more information, see "BCL - .NET Standard 2.0 Support" in the [.NET Framework 4.7.1 Runtime and Compiler Features](https://blogs.msdn.microsoft.com/dotnet/2017/09/28/net-framework-4-7-1-runtime-and-compiler-features) .NET blog.
```

Comment:
.NET blog -> .NET blog post or just blog post or even "post/article on the .NET blog"?